io.github.syntalos.yaml

```diff
@@ -0,0 +1,244 @@
+id: io.github.syntalos
+runtime: org.kde.Platform
+runtime-version: 5.15-23.08
+sdk: org.kde.Sdk
+base: com.riverbankcomputing.PyQt.BaseApp
+base-version: 5.15-23.08
+separate-locales: false
+
+command: syntalos
+rename-icon: syntalos
+rename-desktop-file: syntalos.desktop
+
+finish-args:
+  - --share=network
+  - --share=ipc
+  - --socket=x11
+  - --device=all
+  - --filesystem=host
+  - --socket=system-bus
+  - --socket=session-bus
+  - --allow=devel
+  - --allow=bluetooth
+  - --env=PATH=/app/bin:/usr/bin:/usr/sbin
+  - --env=GST_PLUGIN_SYSTEM_PATH=/app/lib/gstreamer-1.0:/usr/lib/extensions/gstreamer-1.0:/usr/lib/x86_64-linux-gnu/gstreamer-1.0
+  - --require-version=1.12.0
+
+cleanup-commands:
+  - /app/cleanup-BaseApp.sh
+
+add-extensions:
+  org.freedesktop.Platform.GL:
+    directory: lib/GL
+    version: '23.08'
+    subdirectories: true
+    add-ld-path: lib
+    merge-dirs: vulkan/icd.d;glvnd/egl_vendor.d
+    download-if: active-gl-driver
+    enable-if: active-gl-driver
+
+modules:
+- name: essential-dirs
+  buildsystem: simple
+  build-commands:
+    - mkdir -p /app/lib/GL
+
+- name: OpenCL
+  buildsystem: cmake-ninja
+  sources:
+    - type: archive
+      url: https://github.com/KhronosGroup/OpenCL-Headers/archive/v2023.12.14/OpenCL-Headers-v2023.12.14.tar.gz
+      sha256: 407d5e109a70ec1b6cd3380ce357c21e3d3651a91caae6d0d8e1719c69a1791d
+
+- name: meson
+  buildsystem: simple
+  build-commands:
+    - python3 setup.py build
+    - python3 setup.py install --prefix=/app --root=/
+  sources:
+    - type: git
+      url: https://github.com/mesonbuild/meson.git
+      tag: '1.5.1'
+
+- name: xxhash
+  no-autogen: true
+  make-install-args:
+    - PREFIX=${FLATPAK_DEST}
+  cleanup:
+    - /bin/*
+    - /lib/pkgconfig
+    - /lib/*.a
+    - /share
+  sources:
+    - type: archive
+      url: https://github.com/Cyan4973/xxHash/archive/refs/tags/v0.8.2.tar.gz
+      sha256: baee0c6afd4f03165de7a4e67988d16f0f2b257b51d0e3cb91909302a26a79c4
+
+- name: toml++
+  buildsystem: meson
+  builddir: true
+  config-opts:
+    - --libdir=lib
```

Comment:
is this correct? flatpak-builder sets `/app/lib` as default for meson. So this could be not needed.

Reply:
I faintly remember this being required, because Meson otherwise did weird things... But I could remove it and see what happens, this is also fairly old.